org.ekstep.mathtext-1.0/editor/mathtext.js

```diff
@@ -63,103 +63,103 @@ angular.module('org.ekstep.mathtext', [])
     $scope.symbols = {
       greek: [
         {
-          latex: "\\alpha",
```

Comment:
@JagadishPujari Why you added `,` in each object?

Reply:
I have not added, removed that `,` in every object